recipes/mxnet/meta.yaml

```diff
@@ -0,0 +1,147 @@
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - openblas
+    - make
+    - cmake
+    - ninja
+    - ccache
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - python
+    - cython
+    - pip
+    - setuptools
+    - libjpeg-turbo
+    - pkgconfig
+    - pkg-config
+
+  host:
+    # need to better expose these deps
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+    - openblas
+    - mkl  # [(target_platform != "osx-arm64") and (cuda_compiler_version == "None")]
+    - mkl-include  # [(target_platform != "osx-arm64") and (cuda_compiler_version == "None")]
+    - opencv
+    - onednn
+    - cudnn  # [cuda_compiler_version != "None"]
+    - nccl  # [cuda_compiler_version != "None"]
+    - numpy >=1.17
+    - requests >=2.20.0,<3
+    - python-graphviz >=0.8.1,<0.9.0
+    - python_version
+    - libjpeg-turbo
+    - python
+    - pip
+
+  run:
+    # ?
+    - {{ pin_compatible('libgomp') }}  # [linux]
+    - {{ pin_compatible('llvm-openmp') }}  # [osx]
+    - openblas
+    - {{ pin_compatible('mkl') }}  # [(target_platform != "osx-arm64") and (cuda_compiler_version == "None")]
+    - {{ pin_compatible('mkl-include') }}  # [(target_platform != "osx-arm64") and (cuda_compiler_version == "None")]
+    - {{ pin_compatible('opencv') }}
+    - {{ pin_compatible('onednn') }}
+    - {{ pin_compatible('cudnn') }}  # [cuda_compiler_version != "None"]
+    - {{ pin_compatible('nccl') }}  # [cuda_compiler_version != "None"]
+    - {{ pin_compatible('numpy') }}
+    - requests >=2.20.0,<3
+    - python-graphviz >=0.8.1,<0.9.0
+    - python_version
+    - {{ pin_compatible('libjpeg-turbo') }}
+    - python
+    - pip
+
+test:
+  imports:
+    - mxnet
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: http://mxnet.io
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: MXNet is a deep learning framework designed for both efficiency and flexibility
+  description: |
+    Apache MXNet (incubating) is a deep learning framework designed for both
+    efficiency and flexibility. It allows you to mix symbolic and imperative
+    programming to maximize efficiency and productivity. At its core, MXNet
+    contains a dynamic dependency scheduler that automatically parallelizes both
+    symbolic and imperative operations on the fly. A graph optimization layer on
+    top of that makes symbolic execution fast and memory efficient. MXNet is
+    portable and lightweight, scaling effectively to multiple GPUs and multiple
+    machines. MXNet is also more than a deep learning project. It is also a
+    collection of blue prints and guidelines for building deep learning systems,
+    and interesting insights of DL systems for hackers.
+  doc_url: https://mxnet.incubator.apache.org/
+  dev_url: https://github.com/apache/incubator-mxnet
+
+extra:
+  recipe-maintainers:
+    - barry-jin  
```

Comment:
This person should chime in before we can merge. I suggest you add them after in the feedstock itself. instead of now at the beginning.

Reply:
@barry-jin indicated willingness (also a maintainer upstream). See above for the "like" and here https://github.com/conda-forge/staged-recipes/issues/4447#issuecomment-1015956430 but happy to remove for now and tidy up things later. 

Btw, I couldn't figure out a clean way to get the library separated. The python package itself can be noarch and can be built separately. This is one main thing I wanna do once we get this into a feedstock. Another thing is being involved with niche (?) dependencies so that we can avoid "vendoring" them (to be clear, we are not vendoring much, but still I would like to figure out a way to get dlpack, dmlc, nnvm (libtvm) and mshadow from elsewhere.